@daniel-beck 

```
Caused by: java.io.IOException: Failed to load: JUnit Plugin (junit 1.47)
 - Plugin is missing: plugin-util-api (1.5.0)
 - Plugin is missing: bootstrap4-api (4.5.3-1)
 - Plugin is missing: checks-api (1.1.1)
 - Plugin is missing: display-url-api (2.3.4)
 - Plugin is missing: jackson2-api (2.12.0)
 - Plugin is missing: echarts-api (4.9.0-2)
```

--- a/test/src/test/java/hudson/model/AbstractProjectTest.java
+++ b/test/src/test/java/hudson/model/AbstractProjectTest.java
@@ -438,6 +438,16 @@ public class AbstractProjectTest {
         tpm.installDetachedPlugin("scm-api");
         tpm.installDetachedPlugin("workflow-api");
         tpm.installDetachedPlugin("script-security");
+        tpm.installDetachedPlugin("jquery3-api");
+        tpm.installDetachedPlugin("snakeyaml-api");
+        tpm.installDetachedPlugin("jackson2-api");
+        tpm.installDetachedPlugin("popper-api");
+        tpm.installDetachedPlugin("plugin-util-api");
+        tpm.installDetachedPlugin("font-awesome-api");
+        tpm.installDetachedPlugin("bootstrap4-api");
+        tpm.installDetachedPlugin("echarts-api");
+        tpm.installDetachedPlugin("display-url-api");
+        tpm.installDetachedPlugin("checks-api");
         tpm.installDetachedPlugin("junit");
         tpm.installDetachedPlugin("matrix-project");
 

